var/spack/repos/builtin/packages/snpeff/package.py

```diff
@@ -0,0 +1,59 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import os.path
+from shutil import copyfile
+
+
+class Snpeff(Package):
+    """SnpEff is a variant annotation and effect prediction tool. It
+    annotates and predicts the effects of genetic variants (such as
+    amino acid changes)."""
+
+    homepage = "http://snpeff.sourceforge.net/"
+    url      = "https://kent.dl.sourceforge.net/project/snpeff/snpEff_latest_core.zip"
+
+    version('2017-11-24', '1fa84a703580a423e27f1e14a945901c')
+
+    depends_on('jdk', type=('build', 'run'))
+
+    def install(self, spec, prefix):
+        install_tree('snpEff', prefix.bin)
+
+        # Set up a helper script to call java on the jar file,
+        # explicitly codes the path for java and the jar file.
+        script_sh = join_path(os.path.dirname(__file__), "snpEff.sh")
+        script = join_path(prefix.bin, "snpEff")
+        copyfile(script_sh, script)
+        set_executable(script)
+
+        # Munge the helper script to explicitly point to java and the
+        # jar file.
+        java = join_path(self.spec['java'].prefix, 'bin', 'java')
+        kwargs = {'ignore_absent': False, 'backup': False, 'string': False}
```

Comment:
You only change the behaviour for `backup` here. Are you sure this is really what you want? Remove the two non-changed ones